msprime/ancestry.py

```diff
@@ -477,6 +488,17 @@ def simulate(
     :param float length: The length of the simulated region in bases.
         This parameter cannot be used along with ``recombination_map``.
         Defaults to 1 if not specified.
+    :param bool discrete_coordinates: If True, use discrete coordinates
+        in simulations such that recombination breakpoints and mutational
+        sites can only occur at integer positions along the genome.
+        Multiple mutations can occur at the same site.
+        If False (the default) the recombination breakpoints and mutational
+        sites are continuous floating point values. All sites in the
+        returned tree sequence will have exactly one mutation. Please see
+        the :func:`.mutate` function for a more powerful approach to
+        simulating mutations on a tree sequence. It is an error to specify
+        the ``discrete_coordinates`` parameter at the same time as the
+        ``recombination_map`` argument.
```

Comment:
... since this is a property of the recombination_map? Since this also controls mutations maybe it'd be better to allow it to be set but to check if it agrees with the recombination map?

Reply:
oh, I see your note below, now.